Backend/CarRentalApp/CarRentalApp/Services/Data/IUserRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using CarRentalApp.Models.Data;
+
+namespace CarRentalApp.Services.Data
+{
+    public interface IUserRepository
```

Comment:
What do you need repository for?

Reply:
I use this repository as layer between UserService and DbContext